xbmc/storage/discs/IDiscDriveHandler.h

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include <memory>
+#include <string>
+
+/*! \brief Represents the state of a disc (optical) drive
+*/
+enum class DriveState
+{
+  DRIVE_OPEN = 0, ///< The drive is open
+  DRIVE_NOT_READY = 1, ///< The drive is not ready (happens when openning or closing)
+  DRIVE_READY = 2, ///< The drive is ready
+  DRIVE_CLOSED_NO_MEDIA = 3, ///< The drive is closed but no media could be detected in the drive
+  DRIVE_CLOSED_MEDIA_PRESENT = 4, ///< The drive is closed and there is media in the drive
+  DRIVE_NONE = 5, ///< The system does not have an optical drive
+  DRIVE_CLOSED_MEDIA_UNDEFINED =
+      6 ///< The drive is closed but we don't know yet if there's media there
+};
+
+/*! \brief Represents the state of the drive tray
+*/
+enum class TrayState
+{
+  UNDEFINED = 0, ///< The tray is in an undefined state, we don't know yet
```

Comment:
Is the explicit initialization needed?

Reply:
Not needed anymore, came from the translation that was firstly added